modules/openapi-generator/src/main/resources/go/signing.mustache

```diff
@@ -169,6 +174,9 @@ func (h *HttpSignatureAuth) GetPublicKey() (crypto.PublicKey, error) {
 
 // loadPrivateKey reads the private key from the file specified in the HttpSignatureAuth.
 func (h *HttpSignatureAuth) loadPrivateKey() (err error) {
+	if h.privateKey != nil {
+		return nil
+	}
```

Comment:
Do we want to have the condition inside this method? It is not a big deal since it's private, but then we should make it clear in documentation string then it will load it only once (or when `h.privateKey == nil`).

Reply:
Thanks for reviewing the changes. I have made all the requested changes.